microsoft/testsuites/lis/lissuite.py

```diff
@@ -121,97 +121,90 @@ def _verify_lis_preinstall_disk_size(
-                fblock_non_root = stat.get_fs_available_size(lib_module_folder)
-                fblock_root = stat.get_fs_free_blocks(lib_module_folder)
-                boot_part_avail_space = stat.get_fs_block_size(
-                    boot_folder
-                ) * stat.get_fs_available_size(boot_folder)
-                root_part_avail_space = (
-                    stat.get_fs_block_size(lib_module_folder) * fblock_non_root
+            if boot_partition != root_partition:
+                single_partition_file_size_boot = (
+                    boot_part_avail_space - boot_part_required_space
                 )
-                if test_type == "negative":
-                    boot_part_required_space = int(boot_part_required_space / 2)
-                    root_part_required_space = int(root_part_required_space / 2)
-                    # 6.X distro RPM does not use root user space free space.
-                    # Hence setting the min limit.
-                    if fblock_non_root != fblock_root:
-                        root_part_required_space = min_sz_root_partition_not_root
-                    single_partition_file_size = (
-                        root_part_avail_space - root_part_required_space
-                    )
-                    fallocate.create_file(
-                        single_partition_file_size, "/lib/modules/file.out"
-                    )
-                if boot_partition != root_partition:
-                    single_partition_file_size_boot = (
-                        boot_part_avail_space - boot_part_required_space
+                fallocate.create_file(single_partition_file_size_boot, "/boot/file.out")
+            result = lisdriver.install_from_iso()
+            if test_type == "negative":
+                if result.exit_code == 0:
+                    raise LisaException(
+                        "lis should fail to be installed for insufficient space"
                     )
-                    fallocate.create_file(
-                        single_partition_file_size_boot, "/boot/file.out"
-                    )
-                result = lisdriver.install_from_iso()
-                if test_type == "negative":
-                    if result.exit_code == 0:
-                        raise LisaException(
-                            "lis should fail to be installed for insufficient space"
-                        )
-                    else:
-                        log.debug(f"fail to install lis for reason {result.stdout}")
-                self._clean_up_files(node)
-                if test_type == "positive":
-                    lisdriver.uninstall_from_iso()
-            else:
-                raise SkippedException(
-                    f"current distro {node.os.name} doesn't support install LIS"
-                )
+                else:
+                    log.debug(f"fail to install lis for reason {result.stdout}")
+            self._clean_up_files(node)
```

Comment:
This line may not be called, if exception raised above. call it in after_case

Reply:
updated